decidim-core/app/services/decidim/open_data_exporter.rb

```diff
@@ -47,6 +51,18 @@ def data
       buffer.string
     end
 
+    def data_for_resource(resource)
```

Comment:
I am not 100% sure how will this work with the implementation in Open data, which is not using a manifest (at least not yet)